code/setup.dm

```diff
@@ -190,6 +190,9 @@
 #define SLOT_TWOEARS    8192
 #define SLOT_TIE        16384
 #define SLOT_HOLSTER	32768 //16th bit
+#define SLOT_ANY		53247 //Shortcut for allowing an item to fit in any slot. doesnt include denypocket and twoears
```

Comment:
Won't work. Bitfields are limited to 16 bits, this is the 17th one and thus, will not work.